app/actions/websocket/teams.ts

```diff
@@ -49,3 +52,90 @@ export async function handleLeaveTeamEvent(serverUrl: string, msg: any) {
         }
     }
 }
+
+export async function handleUpdateTeamEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    const teamFromData = JSON.parse(msg.data.team);
+    const team: Team = {
+        ...teamFromData,
+    };
+
+    database.operator.handleTeam({
+        teams: [team],
+        prepareRecordsOnly: false,
+    });
+}
+
+export async function handleTeamAddedEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    const user = await queryCurrentUser(database.database);
+    if (!user) {
+        return;
+    }
+
+    const [team, memberships] = await Promise.all([
```

Comment:
I wonder what is the content of the message? Perhaps we should make it so that the websocket event contains the datas that you need and them we avoid going back to the server with a fetch? Maybe time to improve the WS event?

Reply:
`msg` is received as the following: 

```
 { broadcast:
   { team_id: '',
     channel_id: '',
     user_id: '34r7zpnjkbgmpez83xkq1a614c',
     omit_users: null },
  seq: 7,
  data:
   { user_id: '34r7zpnjkbgmpez83xkq1a614c',
     team_id: '5o9abkcuafg59pckjyqhj6ox5w' },
  event: 'added_to_team' }
  ```